apps/www/_blog/2024-08-16-launch-week-12-top-10.mdx

```diff
@@ -47,7 +47,7 @@ Python libs are now officially supported in Supabase. We've seen a huge rise in
 
 ### #5 We released Log Drains
 
-We released Log Drains so that developers can export logs generated by their Supabase products to external destinations, such as Datadog or custom HTTP endpoints.
+We released Log Drains so that developers can export logs generated by their Supabase products to external destinations, such as DataDog or custom HTTP endpoints.
```

Comment:
```suggestion
We released Log Drains so that developers can export logs generated by their Supabase products to external destinations, such as Datadog or custom HTTP endpoints.
```